src/styles.scss

```diff
@@ -856,7 +843,7 @@ compose input.mat-chip-input {
 }
 
 compose .mat-standard-chip {
-    bottom: 6px !important;
+    // bottom: 6px !important;
```

Comment:
I think this commented-out bit can go now :)

Reply:
Right, I was probably checking something out and forgot about it as always.